File: CSharp/SpaceVIL/SpaceVIL/Core/EventTask.cs
using System;
using System.Threading;
using System.Collections.Generic;
using System.Collections.Concurrent;

namespace SpaceVIL.Core
{
    internal sealed class EventTask
    {
        internal InputEventType Action = 0;
        internal Prototype Item = null;
        internal IInputEventArgs Args = null;
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/spacevil/EventTask.java
package com.spvessel.spacevil;

import com.spvessel.spacevil.Core.IInputEventArgs;
import com.spvessel.spacevil.Flags.InputEventType;

final class EventTask {
    InputEventType action = InputEventType.Empty;
    Prototype item = null;
    IInputEventArgs args = null;
}